pkg/util/node.go

```diff
@@ -140,6 +146,28 @@ func RemoveNodeAnnotation(nodeName, key string, client kubernetes.Interface) err
 	return err
 }
 
+func AddNodeLabel(nodeName, key, val string, client kubernetes.Interface) error {
+	var l nodeLabels
+	l.Metadata.Labels = map[string]interface{}{key: val}
+	data, err := json.Marshal(l)
+	if err != nil {
+		return fmt.Errorf("Failed to marshal annotation %v = %v: %s", key, val, err)
+	}
+	_, err = client.CoreV1().Nodes().Patch(nodeName, types.MergePatchType, data)
```

Comment:
I tested it and it worked that way, eg.
```
  labels:
    beta.kubernetes.io/arch: amd64
    beta.kubernetes.io/instance-type: c_c2_m2
    beta.kubernetes.io/os: linux
    ccloud.sap.com/nodepool: payload
    failure-domain.beta.kubernetes.io/region: eu-nl-1
    failure-domain.beta.kubernetes.io/zone: eu-nl-1b
    kubernetes.io/arch: amd64
    kubernetes.io/hostname: kks-jan-payload-s7xbf
    kubernetes.io/os: linux
    kubernikus.cloud.sap/template-version: "6"
    node.kubernetes.io/exclude-from-external-load-balancers: "true"
    node.kubernetes.io/instance-type: c_c2_m2
    topology.cinder.csi.openstack.org/zone: eu-nl-1b
    topology.kubernetes.io/region: eu-nl-1
    topology.kubernetes.io/zone: eu-nl-1b
```
It seems only the label itself get replaced. Otherwise we should have the same problem here I guess: https://github.com/sapcc/kubernikus/blob/master/pkg/util/node.go#L121-L141